components/ui/menu.tsx

```diff
@@ -0,0 +1,54 @@
+import NextLink from 'next/link';
+import { useRouter } from 'next/router';
+
+const routes = [
+  {
+    pathname: '/',
+    routeName: 'all-cats',
+    value: 'All Cats',
+  },
+  {
+    pathname: '/offered',
+    routeName: 'offered',
+    value: 'Offered',
+  },
+  {
+    pathname: '/my-cats',
+    routeName: 'my-cats',
+    value: 'My Cats',
+  },
+  {
+    pathname: '/my-requests',
+    routeName: 'my-requests',
+    value: 'My requests',
```

Comment:
this doesn't matter much, but all the other values are first letter capitalised. So this would be 'My Requests'